rotkehlchen/premium/premium.py

```diff
@@ -296,12 +296,19 @@ def query_last_data_metadata(self) -> RemoteMetadata:
             raise RemoteError(msg) from e
 
         result = _process_dict_response(response)
```

Comment:
should we check the status code here? The server returned a 404 that is allowed in this function but doesn't raise any error

Reply:
ah good point. Will add it.